subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -1006,20 +1024,27 @@ static void isr_rx(void *param)
 	/* Encryption */
 	if (lll->enc) {
 		uint64_t payload_count;
+		struct pdu_bis *pdu;
 
 		payload_count = lll->payload_count - lll->bn;
 		if (bis) {
+			static uint8_t big_ctrl_pdu[offsetof(struct node_rx_pdu, pdu) +
+						    sizeof(struct pdu_big_ctrl)];
```

Comment:
It's not something we do alot (but perhaps should): Using `offsetof` is a nice way to do something like this, but it is very fragile. Any changes to struct (e.g. adding or removing a field) may break the assumption here. 

One trivial approach is to of course to add a comment in the struct that this should not be modified without considering the sideeffects. 

We could probably also use a `BUILD_ASSERT` with calls to `offsetof` to help ensure correctness.

Reply:
offsetof was not needed, it was my confusion on buffer size to be allocated. Only the actual max control PDU length buffer is that all needed, now this dedicated buffer is used for both broadcaster and sync receiver.